README.md

```diff
@@ -41,120 +56,13 @@ You can find an overview over the features available in Alacritty [here](./docs/
 
 ## Installation
 
-Some operating systems already provide binaries for Alacritty, for everyone
-else the instructions to build Alacritty from source can be found [here](INSTALL.md).
-
-### Alpine Linux
-
-```sh
-apk add alacritty
-```
-
-### Arch Linux
-
-```sh
-pacman -S alacritty
-```
-
-### Fedora
-
-Unofficial builds of stable tags can be found in Fedora Copr:
-[pschyska/alacritty](https://copr.fedorainfracloud.org/coprs/pschyska/alacritty/).
-
-``` sh
-dnf copr enable pschyska/alacritty
-dnf install alacritty
-```
-
-If you want to help test pre-releases, you can additionally enable
-[pschyska/alacritty-testing](https://copr.fedorainfracloud.org/coprs/pschyska/alacritty-testing/).
-
-### Gentoo Linux
-
-```sh
-emerge x11-terms/alacritty
-```
-
-### GNU Guix
-
-```sh
-guix package -i alacritty
-```
-
-### Mageia
-
-```sh
-urpmi alacritty
-```
-
-### NixOS
-
-```sh
-nix-env -iA nixos.alacritty
-```
-
-### openSUSE Tumbleweed
-
-```sh
-zypper in alacritty
-```
-
-### Pop!\_OS
-
-```sh
-apt install alacritty
-```
-
-### Solus
-
-```sh
-eopkg install alacritty
-```
-
-### Void Linux
-
-```sh
-xbps-install alacritty
-```
-
-### FreeBSD
-
-```sh
-pkg install alacritty
-```
-
-### macOS
-
-```sh
-brew cask install alacritty
-```
-
-Once the cask is installed, it is recommended to setup the manual page, shell
-completions, and terminfo definitions. These are located inside the installed
-application's Resources directory: `Alacritty.app/Contents/Resources`.
-
-### Windows
-
-#### Via [Chocolatey](https://chocolatey.org)
-
-```batch
-choco install alacritty
-```
+### Manual installation
+For everyone else the instructions to build Alacritty from source can be found [here](INSTALL.md).
```

Comment:
This wording doesn't make sense anymore. I also don't think there's a point to have two subsections for essentially just two paragraphs.

So we can just have one sentence about prebuilt binaries and then a second paragraph talk about building from source. In the prebuilt binary section we should also mention that Alacritty is available on various package managers on Linux, BSD, macOS and Windows. That should encourage people to search their respective repos.

Reply:
Okay, I will update later on.